financial-connections/src/main/java/com/stripe/android/financialconnections/features/consent/ConsentPreviewParameterProvider.kt

```diff
@@ -122,6 +123,7 @@ internal class ConsentPreviewParameterProvider :
             " We never share your login details with them.",
         cta = "Agree",
         dataAccessNotice = DataAccessNotice(
+            icon = Image("https://www.cdn.stripe.com/12321312321.png"),
```

Comment:
Are these supposed to be live URLs? We should try to avoid them so that the tests can’t be flaky.

Reply:
`StripeImage` just renders the `debugResource` drawable if on preview mode, so these links are basically ignored, but we can pass a resource instead just in case